bin/ns-status.js

```diff
@@ -25,30 +24,26 @@ var requireUtils = require('../lib/require-utils')
 */
 
 function mmtuneStatus (status) {
-    if (mmtune_input) {
-        var mmtune = requireWithTimestamp(cwd + '/' + mmtune_input);
-        if (mmtune) {
-            if (mmtune.scanDetails && mmtune.scanDetails.length > 0) {
-                mmtune.scanDetails = mmtune.scanDetails.filter(function (d) {
-                    return d[2] > -99;
-                });
-            }
-          status.mmtune = mmtune;
+    var mmtune = requireWithTimestamp(cwd + mmtune_input);
+    if (mmtune) {
+        if (mmtune.scanDetails && mmtune.scanDetails.length) {
```

Comment:
Why are you removing the `> 0` here?

Reply:
It's redundant. If length is truthy, then it's value must be greater than 0